microsoft-edge/web-platform/release-notes/135.md

```diff
@@ -0,0 +1,462 @@
+
+
+<!-- ---------- -->
+###### Remove `navigator.xr.supportsSession` method
+
+The deprecated `navigator.xr.supportsSession` property has been removed from the WebXR API.
+
+In the WebXR specification in 2019, the `navigator.xr.supportsSession` property was marked as deprecated, and was replaced by the `navigator.xr.()` method.
+
+
+<!-- ---------- -->
+###### Remove WebGPU limit `maxInterStageShaderComponents`
+
+The `maxInterStageShaderComponents` limit is now removed, due to:
+
+* Redundancy with `maxInterStageShaderVariables`, which already serves a similar purpose: controlling the amount of data that's passed between shader stages.
+
+* Minor discrepancies: Although there are slight differences in how the two limits are calculated, these differences are minor, and can be effectively managed within the `maxInterStageShaderVariables` limit.
+
+* Simplification: Removing `maxInterStageShaderComponents` streamlines the shader interface, and reduces complexity of using the interface.  Instead of managing two separate limits (that both apply simultaneously, but with subtle differences), you can just concentrate on the more appropriately named and comprehensive `maxInterStageShaderVariables`.
+
+
+<!-- ====================================================================== -->
+## Origin trials
+
+The following are new experimental APIs which you can try on your own live website for a limited time.
+
+To learn more about origin trials, see [Use origin trials in Microsoft Edge](../../origin-trials/index.md).
+
+To see the full list of available origin trials, see [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials/).
+
+
+<!-- ------------------------------ -->
+#### Microsoft Edge-only origin trials
+
+
+<!-- ---------- -->
+###### Digital Goods API
+
+Expires on March 31, 2025.
+
+The Digital Goods API allows a web application to get information about their digital products and the user’s purchases managed by a digital store.  The user agent abstracts connections to the store, and the Payment Request API is used to make purchases.
+
+* [Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/PwaDigitalGoods/explainer.md)
+* [Feedback](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/new?labels=Digital+Goods%20API,OriginTrialFeedback&amp;title=%5BDigital%20Goods%20API%5D+Feedback)
+* [Register](https://developer.microsoft.com/en-us/microsoft-edge/origin-trials/trials/4b4a9ead-d912-4349-87b3-25e5e50b4f13)
+
+
+
+<!-- ---------- -->
+###### Web App LocalFolder Access
+
+Expires on March 31, 2025.
+
+Web App LocalFolder Access allows a Microsoft Store-installed Progressive Web App (PWA) to access file content that was previously stored in the WinRT `ApplicationData.LocalFolder` folder by an earlier,<!-- todo: analyze implication of comma vs no comma --> UWP version of the application.
```

Comment:
"earlier, UWP version of the application" -- confirm whether to use comma.  or clarify.  

Which one gives clearest meaning?  Can we increase clarity to specify which thing?
* to access file content that was previously stored in the WinRT `ApplicationData.LocalFolder` folder by an earlier UWP version of the application.
* to access file content that was previously stored in the WinRT `ApplicationData.LocalFolder` folder by an earlier, UWP version of the application.

Are we switching from a UWP version of app to a non-UWP version of app?  If so, use comma.
Is the earlier & updated version of the app both UWP?  If so, omit comma, maybe move the word "UWP" out.